### PR TITLE
Update opcodes and version for KR 6.3

### PR DESCRIPTION
**Pull request by Ra-Workspace**

### CODE REVIEW

**Comment by Ra-Workspace** [comment]
Thanks to Wintermute for helping get the opcodes.

File: OverlayPlugin.Core/resources/opcodes.jsonc [hunk 1]
```diff
@@ -380,5 +380,21 @@
...
+            "size": 11
+        },
+        "CEDirector": {
+            "opcode": 777,
+            "size": 16
+        },
+        "RSVData": {
+            "opcode": 115,
+            "size": 1080
+        }
```
Comment (quisquous) [inline OverlayPlugin.Core/resources/opcodes.jsonc:398]: Is this backwards? Are these the KR opcodes and you didn't mean to change the global ones?

Comment (Ra-Workspace) [reply OverlayPlugin.Core/resources/opcodes.jsonc]: It's obviously my mistake. Maybe I dozed off late at night.

**Comment by Ra-Workspace** [comment]
I'm sorry I didn't check it properly. I fixed it

**Pull request merged**